enterprise-suite/scripts/lbc.py

```diff
@@ -101,7 +107,7 @@ def run(cmd, timeout=None, stdin=None, show_stderr=True):
 # prints it to stdout or stderr, handles failure status
 # codes by exiting with and error if can_fail=False.
 def execute(cmd, can_fail=False, print_to_stdout=False):
-    printerr(cmd)
+    printinfo(cmd)
```

Comment:
This should be `printerr`, so it goes to `stderr`. Idea is to be able to pipe output to a useful file `lbc.py install --export-yaml=console > console.yaml`

Reply:
That seems like a bug, why did you approve the PR? :)

Reply:
ok got it..